app/services/MultiTenantService.rb

```diff
@@ -0,0 +1,15 @@
+class MultiTenantService
+  attr_accessor :service_org
```

Comment:
If this isn't going to be accessible outside of the class you can put this beneath private so the surface area of the class stays tiny. I found it good to keep the surface area small so things stay manageable.

```
private

attr_accessor :service_org
```